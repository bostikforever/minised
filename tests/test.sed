:loop
 s/\(.*[0-9]\)\([0-9]\{3\}\)/\1,\2/
 t loop
