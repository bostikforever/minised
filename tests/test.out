12,342,342,342,343,434
